src/api/addressApi.js

```diff
@@ -18,26 +18,33 @@ const addressApi = {
     });
   },
 
-  search(address, count, hash, page, token) {
+  getBalance(address, token) {
+    return requestExplorerServiceV1.get(`address/balance`, {params: {address, token}}).then((res) => {
+      if (res && res.data) {
+        return res.data
+      }
+    }).catch((error) => {
+      // something wrong with request
+    });
+  },
+
+  getHistory(address, token, count, skip) {
     /*
      address: address to search
-     count: int -> how many objects we want
-     hash (optional): str -> hash reference for the pagination (works together with 'page' parameter)
-     page (optional): 'previous' or 'next' -> if 'previous', we get the objects before the hash reference
-                                   if 'next', we get the objects after the hash reference
-     token (optional): str -> only fetch txs related to this token uid
+     token: str -> only fetch txs related to this token uid
+     count (optional): int -> how many objects we want
+     skip (optional): str -> skip this many transactions before fetching
```

Comment:
This is similart o a `offset`?

Reply:
Exactly

Reply:
Changed to limit/offset to be more intuitive.